CMakeLists.txt

```diff
@@ -7,6 +7,12 @@ endif()
 # See docs/release_checklist.md
 project(SDL3 LANGUAGES C CXX VERSION "3.0.0")
 
+if (CMAKE_SYSTEM_NAME MATCHES "visionOS")
+  set(VISIONOS ON)
+else()
+  set(VISIONOS OFF)
+endif()
+
```

Comment:
Please move this to [here](https://github.com/libsdl-org/SDL/blob/8adab0b88ab1c0aa69242fffc48afdba085b1dc4/cmake/sdlplatform.cmake#L8-L17).